CTFd/challenges.py

```diff
@@ -36,7 +36,7 @@ def chals():
             if view_after_ctf():
                 pass
             else:
-                return redirect('/')
+                return redirect(url_for('/'))
```

Comment:
Not sure this is valid, but if you specify the root in `config.py` you should be able to pull that value and redirect to it.